app/src/main/java/xyz/jhughes/laundry/LaundryParser/Machine.java

```diff
@@ -13,6 +13,8 @@
     private String status;
     private String time;
 
+    // TODO: Call constructor when creating machines so stripping leading 0s doesn't
```

Comment:
I think these are created automatically with Retrofit, so either "setName()" or the constructor can be trusted to be called every time. I am not completely sure which though.